src/deezer/pagination.py

```diff
@@ -92,10 +99,10 @@ async def _fetch_to_index(self, index: int):
 
     async def fetch(self) -> PaginatedList:
         """Function to fetch the thing"""
-        try:
+        if hasattr(self.__base_params, "limit"):
             limit = int(self.__base_params["limit"])
-        except KeyError:
-            limit = 500
```

Comment:
if `__base_params` is a dict, why don't we jut use `self.__base.get("limit", 99999)`?

Reply:
Yeah i didn't even know you could do that